concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/10 Configure Views.md

```diff
@@ -0,0 +1,117 @@
+The **Scheduler** supports the following view types:
+
+* [Day](/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Day_View)
+
+* [Week and Work week](/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Week_and_WorkWeek_Views)
+
+* [Month](/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Month_View)
+
+* [Timeline Day, Month, Week, and Work week](/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Timeline_Views)
+
+* [Agenda](/Documentation/Guide/UI_Components/Scheduler/Views/View_Types/#Agenda_View)
+
+Use the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array to configure views. This array can contain strings ([view types](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type)) and objects (view configurations). Use a string if the default view configuration suits you. Otherwise, declare an object that configures the view and specify the [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and [other properties](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) in it. In the code below, the **views[]** array contains the Day, Week, and Month views. We specify the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties for Day and Week; Month uses its default configuration.
```

Comment:
```suggestion
Use the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array to configure views. This array can contain strings ([view types](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type)) and objects (view configurations). Use a string if the default view configuration suits you. Otherwise, declare an object that configures the view and specify the [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and [other properties](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) within it. In the code below, the **views[]** array contains the Day, Week, and Month views. We specify the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties for Day and Week; Month uses its default configuration.
```